version.go

```diff
@@ -0,0 +1,3 @@
+package godatabend
+
+var version = "0.4.2"
```

Comment:
Maybe wee could use `debug.ReadBuildInfo()`?

Reply:
Sounds nice!

Reply:
The debug.readBuildInfo if suit for build binary proj since it must use `ldflags -X main.Version` flag to build. This module was imported and maybe not convenient to get version from `debug.readBuildInfo`.